libafl_cc/src/cmplog-switches-pass.cc

```diff
@@ -233,7 +166,7 @@ bool CmpLogSwitches::hookInstrs(Module &M) {
   }
 
   for (auto &F : M) {
-    if (!isIgnoreFunction(&F)) { continue; }
```

Comment:
@tokatoka I reversed this not, it seems it was broken, can you check?

Reply:
yes you are correct